sf_stats.Api/sf_stats.DataAccess.MSSql/Context/SFStatDbContext.cs

```diff
@@ -19,7 +19,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
```

Comment:
I think you forgot to add the `public DbSet<Season> Season { get; set; }` for each of the entities you added

Reply:
added